mri/operators/linear/test_wavelet_adjoint.py

```diff
@@ -107,7 +107,11 @@ def test_Wavelet_UD_2D_Multichannel(self):
         multichannel case"""
         for i in range(self.max_iter):
             print("Process Wavelet Undecimated test '{0}'...", i)
-            wavelet_op = WaveletUD2(nb_scale=4, multichannel=True, n_cpu=2)
+            wavelet_op = WaveletUD2(
+                nb_scale=4,
+                n_coils=self.num_channels,
+                n_jobs=2
```

Comment:
nitpicking here, but with a comma at the end you would even prevent unnecessary comma addition when you have a new argument to add

Reply:
Hmm i didnt know that was allowed. But that would be a huge change, will just keep in mind for furture.